apps/website/docs/api/01-babel/01-preset.md

```diff
@@ -0,0 +1,73 @@
+---
+title: preset
+slug: /api/babel-preset
+---
+
+# Babel preset
+
+<p className="text-xl">React Strict DOM includes a Babel preset to generate and optimize web app builds.</p>
+
+:::info[Web only]
+
+Configuring Babel to use this preset is required for web builds only. It should not be used for native builds.
+
+:::
+
+## Overview
+
+The preset ensures that production builds have **no runtime performance overhead on web** relative to using [React DOM](https://react.dev/) with an advanced atomic CSS solution like [StyleX](https://stylexjs.com). The preset is required to extract styles to static CSS and optimize element rendering.
```

Comment:
```suggestion
The preset ensures that production builds have **no additional runtime performance overhead on web** relative to using [React DOM](https://react.dev/) with an advanced atomic CSS solution like [StyleX](https://stylexjs.com). The preset is required to extract styles to static CSS and optimize element rendering.
```

Reply:
"additional" is redundant given it says "relative to using..."

Reply:
Yeah, I think the language can be a bit clearer even if the word "additional" *also* sounds a bit awkward.